docs: Update documentation for controller PR #2059 (#107)
cartridge-gg/docs: src/pages/controller/getting-started.mdx

}
```

> ## Wallet Standard Integration
> 
> The ControllerConnector supports the [get-starknet wallet standard](https://get-starknet.vercel.app/), enabling integration with frontend libraries like `starknet-react` and generic libraries like `solid.js`. This provides a higher-level interface than the standard `StarknetWindowObject`.
> 
> ### Using asWalletStandard()
> 
> You can convert a ControllerConnector to a wallet-standard compatible interface:
> 
> ```tsx
> import { ControllerConnector } from "@cartridge/connector";
> import type { WalletWithStarknetFeatures } from "@starknet-io/get-starknet-wallet-standard/features";
> 
> const controller = new ControllerConnector({
>   policies: {
>     // Your session policies
>   }
> });
> 
> // Convert to wallet standard interface
> const walletStandard: WalletWithStarknetFeatures = controller.asWalletStandard();
> 
> // Now compatible with any library that expects WalletWithStarknetFeatures
> ```
> 
> :::warning
> The `asWalletStandard()` method is an experimental feature and may contain bugs. Please [report any issues](https://github.com/cartridge-gg/controller/issues) you encounter.
> :::
> 

Reviewer: ```suggestion
## Wallet Standard Integration

The ControllerConnector supports the [get-starknet wallet standard](https://get-starknet.vercel.app/), a higher-level interface than the`StarknetWindowObject` which enables alternative integration with libraries like `starknet-react` and `solid.js`.

### Using asWalletStandard()

You can use ControllerConnector to cast Controller to the wallet-standard compatible interface `WalletWithStarknetFeatures`

```tsx
import { ControllerConnector } from "@cartridge/connector";

const controllerConnector = new ControllerConnector();
const controllerAsWalletStandard = controllerConnector.asWalletStandard() // type WalletWithStarknetFeatures
```

:::warning
This is an experimental feature and may contain bugs.
:::

```

> ### Alternative Direct Casting

Reviewer: ```suggestion
```
